src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -274,7 +289,7 @@ def load_arguments(self, _):
         c.argument('disable_scope_strict_match', options_list=['--disable-scope-strict-match', '-i'], action='store_true', help='Include policy exemptions either inherited from parent scope or at child scope.')
         c.argument('display_name', help='Display name of the policy exemption.')
         c.argument('description', help='Description of policy exemption.')
-        c.argument('exemption_category', options_list=['--exemption-category', '-e'], help='The policy exemption category of the policy exemption', arg_type=get_enum_type(ExemptionCategory))
```

Comment:
Also removed this, so the removal of the import wouldn't cause another error

Reply:
There is no need to remove this line if adopt https://github.com/Azure/azure-cli/pull/24211#discussion_r1240997608

Reply:
As stated in this [comment ](https://github.com/Azure/azure-cli/pull/24211#discussion_r1241468772) this fix does not work. The import causes more test failures.

Reply:
Hi @harshpatel17
since it is mentioned that the ExemptionCategory is only in a preview folder but not latest stable folder (https://github.com/Azure/sdk-release-request/issues/4179#issuecomment-1579765510), to list the available values manually is also feasible. You don't have to import the class from the sdk.